it/uk/gov/hmrc/transitmovementsrouter/controllers/MessagesControllerIntegrationSpec.scala

```diff
@@ -139,7 +143,7 @@ class MessagesControllerIntegrationSpec
           urlEqualTo("/gb")
         )
           .withHeader(HeaderNames.AUTHORIZATION, equalTo(s"Bearer bearertoken"))
-          .withHeader("Date", equalTo(formatted))
+          .withHeader("Date", anything)
```

Comment:
Hmm, I'm not so sure about this -- the spec does state that we need to have it in a certain format which is what was specified. I was lazy in some cases, but I think this was one that tested that the date we injected was correct.

Why did you change it?